web-ui/src/main/resources/catalog/components/edit/onlinesrc/OnlineSrcService.js

```diff
@@ -483,21 +483,15 @@
         removeThumbnail: function(thumb) {
           var scope = this;
 
-          // It is a url thumbnail
-          if (thumb.id.indexOf('resources.get') < 0) {
+          if(thumb && gnCurrentEdit.id) {
```

Comment:
This will remove the reference to the file/url in the metadata XML but will not remove local uploaded files from the files storage.  The user will need to open the _Add resource online` dialog and click the remove button next to the file to remove it physically from the system.